presto-main/src/main/java/io/prestosql/memory/NodeMemoryConfig.java

```diff
@@ -31,7 +31,7 @@
     public static final String QUERY_MAX_MEMORY_PER_NODE_CONFIG = "query.max-memory-per-node";
     public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE_CONFIG = "query.max-total-memory-per-node";
 
-    private boolean isReservedPoolDisabled;
+    private boolean isReservedPoolDisabled = true;
```

Comment:
should we also make the property `@depricated?`

Reply:
Good idea. Done.

Reply:
It seems the annotations for getters are missing. 
```
1) Error: Methods [public io.prestosql.memory.NodeMemoryConfig io.prestosql.memory.NodeMemoryConfig.setReservedPoolDisabled(boolean)] and [public boolean io.prestosql.memory.NodeMemoryConfig.isReservedPoolDisabled()] must be @Deprecated together
```